libfwupd/fwupd-client.c

```diff
@@ -121,13 +123,116 @@ fwupd_client_curl_helper_free (FwupdCurlHelper *helper)
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(FwupdCurlHelper, fwupd_client_curl_helper_free)
 #endif
 
+typedef struct {
+	FwupdClient	*self;
+	gchar		*property_name;
+	guint		 signal_id;
+	FwupdDevice	*device;
+} FwupdClientContextHelper;
+
+static void
+fwupd_client_context_helper_free (FwupdClientContextHelper *helper)
+{
+	if (helper->device != NULL)
+		g_object_unref (helper->device);
```

Comment:
Could simplify this with `g_clear_object (&helper->device)`

Reply:
Fixed, ta.